nengo_bones/templates/docs.sh.template

```diff
@@ -5,7 +5,7 @@
 {% block install %}
 {{ super() }}
     exe pip install "sphinx>=1.8.0" "jupyter>=1.0.0" "nbsphinx>=0.2.13" "nengo-sphinx-theme>=0.7.0" "numpydoc>=0.6.0"
-    exe pip install -e ".[docs]"
+    exe pip install -e ".[doc]"
```

Comment:
Why did we change these to `doc` and `test` instead of `docs` and `tests`? I think up this points we've been using the plural, so switching to singular would mean changing documentation if we recommend using these somewhere, and will be confusing if someone was previously using one of these and then has to switch.

Reply:
I changed the variables in `setup.py` to `doc_req`, `test_req` etc., because I always found it confusing keeping track of which were plural and which weren't (e.g. `install_requires` vs `docs_require` vs `optional_requires`).  So I standardized on the singular, since that seemed less grammatically awkward (e.g. `installs_req`). And I changed from `requires` to `req` to make it feel less like a normal word that we'd expect to be pluralized one way or the other.  And then since those were all changing to singular, it seemed like it might be confusing to have the install names be pluralized.  I didn't expect anyone to be using these other than developers, so I didn't think there'd be too much user impact from swapping.  But we could also leave it as plural, and not worry about the mismatch between `[docs]` and `doc_req` (which people might not even notice or be confused by, unless they're inspecting the `setup.py` code).

Reply:
The reason it's done with singular/plurals in those spots is to be consistent with `setuptools` itself, as the `setup` function accepts `install_requires`, `tests_require` and `extras_require` keyword arguments.

But we're kind of free to do what we want for our own naming things as long as we pass things to the correct keyword argument, so I'm fine with `_req` instead of `_require(s)`. For `doc`/`test` vs `docs`/`tests`, however, I would rather be consistent with other places in our source tree; e.g., we have a top-level `docs` directory, not a `doc` directory, and within `nengo_bones` and other places we have `tests` directories, not `test`, so I would find it confusing to switch to singular here.